docs/documentation/mission_module/mission_file/segments.rst

```diff
@@ -433,10 +433,30 @@ Typically, it will be used as last segment to compute a reserve based on the Zer
       altitude: 0.
       mach: 0.
 
+Mass loss without fuel consumption
+----------------------------------
+
+.. versionadded:: 1.8.2
+
+Using :code:`delta_mass` allows to simulate a fuel consumption equivalent to the mass loss.
+
+For cases where mass loss should be simulated without fuel consumption, it is possible to set to
+:code:`False` the parameter :code:`fuel_is_consumed`.
+
+**Example:**
+
+.. code-block:: yaml
+
+    segment: transition
+    target:
+      delta_mass: 100.
```

Comment:
On the example the delta_mass is postive --> mass gain instead of mass loss, it's not wrong but wouldn't it be more clear as an example if the dmass was negative?(as it was done at the test of the function)

Reply:
Applied in fcc948b001231f531f68d73d034a466598b4bb0f